getting-started/contributing.md

```diff
@@ -0,0 +1,30 @@
+# Contributing
+
+WDL is a community project and, thus, depends on _your_ contributions to make sure it
+continues to thrive. Contributions can come in various forms, including
+
+- filing issues when you run into problems on GitHub,
+- participating as part of language design discussions (via the [RFC process]),
+- contributing language or code changes via [pull requests], and
+- helping answer questions on [Slack][slack-invite].
+
+Please browse the [OpenWDL organization] to learn about the different parts of the
+project and get an idea of how you could contribute. Keep in mind that contributions to
+WDL must be contributed according to the `CONTRIBUTING.md` ([link][contributing-md])
+document.
+
+### Suggested first steps
+
+While all of these projects are important, consider starting with the [specification]
```

Comment:
Do these need links?

Reply:
Yes! It automatically links to the ones below if the name in brackets matches the prefixes below exactly. So this should work properly.